olca-app/src/org/openlca/app/editors/processes/data_quality/DataQualityCell.java

```diff
@@ -59,12 +64,21 @@ void createComponents(Composite parent, FormToolkit toolkit) {
 		label = toolkit.createLabel(composite, text, SWT.WRAP);
 		label.addMouseTrackListener(new MouseOver());
 		label.addMouseListener(new MouseClick());
+		label.addPaintListener(new PaintListener() {
+			@Override
+			public void paintControl(PaintEvent e) {
+				setColor();
+				label.removePaintListener(this);
+			}
+		});
 	}
 
 	private class MouseOver implements MouseTrackListener {
 		@Override
 		public void mouseEnter(MouseEvent e) {
-			Color color = DQUI.getColor(score.position, indicator.scores.size());
+			Color color = Objects.requireNonNullElse(
```

Comment:
It seems that `Colors.background()` is always the default color, then `DQUI.getColor` could directly return it and we do not need this long expression always.